SU-SWS/stanford_faq
# Tagging 7.x-4.3-dev to 7.x-4.3 and fixing version and deprecated term…

# READY FOR REVIEW

# Summary
- Tagging new release and updating CHANGELOG and .codeclimate.yml settings.

# Needed By (Date)
- Needed when it can be done.

# Urgency
- 3/10

# Steps to Test

1. Checkout `7.x-4.3-tagging`
2. Run `codeclimate validate-config` and confirm there are no errors or warnings.
3. Review Code
4. Merge

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/SOE-3381

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
